plugin.go

```diff
@@ -0,0 +1,89 @@
+// Package plasmactlupdate implements an update launchr plugin
+package plasmactlupdate
+
+import (
+	"github.com/launchrctl/keyring"
+	"github.com/launchrctl/launchr"
+	"github.com/launchrctl/launchr/pkg/log"
+	"github.com/skilld-labs/plasmactl-update/update"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	launchr.RegisterPlugin(&Plugin{})
+}
+
+// Plugin is launchr plugin providing update action.
+type Plugin struct{}
+
+// PluginInfo implements launchr.Plugin interface.
+func (p *Plugin) PluginInfo() launchr.PluginInfo {
+	return launchr.PluginInfo{}
+}
+
+// OnAppInit implements launchr.Plugin interface.
+func (p *Plugin) OnAppInit(_ launchr.App) error {
+	return nil
+}
+
+// CobraAddCommands implements launchr.CobraPlugin interface to provide bump functionality.
+func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
+	var creds keyring.CredentialsItem
+
+	var updCmd = &cobra.Command{
+		Use:   "update",
+		Short: "Command to fetch and install latest version of plasmactl",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+			u, err := update.CreateUpdate(creds)
+			if err != nil {
+				return err
+			}
+
+			return u.RunUpdate()
```

Comment:
Probably better to keep same pattern as other plugins where this command executes the function bellow (line 57)

Then top level logic lives in this function, which can call some help function from `update.go` if needed